### DISCUSSION TITLE
Haven't been approved after getting veriified

### DISCUSSION THREAD

**Discussion by saintlypioneer** [community/community#133943] [GitHub Education]
### Hi everyone,

I have been trying to verify my student membership and after some attempts, I could do that but, although several days have passed, it still shows that I will receive the benefits **soon**!

Do I need to reverify or something? Can anyone please guide me here?

**Comment by davevad93** [comment]
Hi @saintlypioneer ,

Don't worry, you've been approved and don't have to reverify, just wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @saintlypioneer was approved and your benefits are now available.`

For further reference you can look at the official [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And take a look at the following similar discussions also: #132790 , #131665 , #132470

**Comment by Sivamani-18** [comment]
verification process for your student membership but haven't yet received the benefits. Here are some steps you can take to resolve the issue:

1. **Check Verification Status**: Log into your account on the platform where you applied for the student membership and check if there are any updates or notifications regarding your verification status.

2. **Review Confirmation Email**: If you received a confirmation email when you completed the verification process, review it to ensure that it doesn't contain any additional steps you might have missed.

3. **Contact Support**: Reach out to the customer support team of the service you're trying to get the student benefits from. They can provide specific information about the status of your verification and why there might be a delay.

4. **Re-verification**: If you suspect that the verification process might not have completed correctly, you can try re-submitting your verification documents. However, make sure to check with support first to avoid any confusion or duplicate submissions.

5. **Check Spam/Junk Folder**: Sometimes important emails can end up in the spam or junk folder. Make sure to check there for any communication regarding your student membership.
